recipes/core/cronie.yaml

```diff
@@ -0,0 +1,47 @@
+# SUMMARY = "Cron daemon for executing programs at set times"
+# DESCRIPTION = "Cronie contains the standard UNIX daemon crond that runs \
+#                specified programs at scheduled times and related tools. \
+#                It is based on the original cron and has security and \
+#                configuration enhancements like the ability to use pam \
+#                and SELinux."
+# HOMEPAGE = "https://github.com/cronie-crond/cronie/"
+
+inherit: [cpackage, pkg-config, make, install]
```

Comment:
This seems to be a standard autotools based package. Unless something is really off, the `autotools` class should work.